content/docs/faq.md

```diff
@@ -11,27 +11,64 @@ embed_js:
+
+## System Requirements
+
+### What are the system requirements for Scratch Addons?
+
+Scratch Addons is officially supported on [Google Chrome](https://google.com/chrome/) (version 80 and above) and [Mozilla Firefox](https://mozilla.org/firefox/) (version 74 and above). Scratch Addons should also work on other desktop browsers that are based on those browsers, such as:
+
+- [Microsoft Edge](https://microsoft.com/edge/) (version 80+)
+- [Opera](https://opera.com/download/) (version 67+)
+- [Vivaldi](https://vivaldi.com/download/) (version 2.11+)
+- [Brave](https://brave.com/download/) (version 1.3+)
+
+Additionally, Scratch Addons (for obvious reasons) requires Internet connection.
+
+### Can I use Scratch Addons on a mobile device?
+
+For Android users: Yes, but it is not recommended. Major browsers do not allow Scratch Addons (or any other extensions) to be installed, so you need to use browsers such as [Kiwi](https://kiwibrowser.com/) to do so. Scratch Addons' UI is not well-tested on touchscreens or environments with small viewport size, so some of the features might not work as expected.
+
+For iOS and iPadOS users: Sadly, it is not. App Store policy does not allow browser implementations to be uploaded, which means all browsers available on that platform are just re-skinned Safari. This causes some problems (see below).
+
+### Can I use Scratch Addons on Safari?
+
+Currently, you cannot.
+
+First, Safari extension store requires all developers to pay an annual fee to list extensions on the store. As Scratch Addons team does not have a source of income, this makes it very hard to maintain the extension. There is also a technical problem with the implementation of browser extensions in Safari which makes some of the core features to be unusable.
+
+### Can I use Scratch Addons on the offline editor?
+
+Most of the project editor addons are also available on [TurboWarp](https://turbowarp.org/) which has a [downloadable app](https://desktop.turbowarp.org/) for Windows, macOS, and Linux. Additionally, on browsers that support Progressive Web Application (PWA) such as Google Chrome on Chromebook, you can also install the TurboWarp editor as a PWA and use it without Internet connection.
+
+Scratch Addons cannot be used on the official Scratch app.
+
+### Are there any incompatible programs?
+
+Some browser extensions and userscripts may interfere with Scratch Addons. If you experienced problems, you should turn off those programs:
+
+- Scratch 3 Developer Tools: This browser extension is a copy of the Developer tools addon. You should uninstall the Developer Tools browser extension and turn on the addon instead.
+- Better3.0: This browser extension can interfere with some addons. Luckily, most of its features are also available as addons.
+- Redux DevTools: This can interfere with the internal workings of Scratch Addons. Disable it if you are not using.
```

Comment:
The last sentence of this line sounds off

Reply:
I'll fix it